bus/routes.go

```diff
@@ -181,6 +181,26 @@ func (b *Bus) consensusNetworkHandler(jc jape.Context) {
 	})
 }
 
+func (b *Bus) postSystemSQLite3BackupHandler(jc jape.Context) {
```

Comment:
`nit` inconsistent with route name

Reply:
How so? Route name is `POST /system/sqlite3/backup`